src/IR/IR.hs

```diff
@@ -252,3 +255,97 @@ wellFormed (Prim p es _) = wfPrim p es && all wellFormed es
   wfPrim (PrimOp PrimLt    ) [_, _]    = True
   wfPrim (PrimOp PrimLe    ) [_, _]    = True
   wfPrim _                   _         = False
+
+
+{-
+how to use:
```

Comment:
These command line usage notes should be removed from documentation inline, esp the questions further below.